test/__init__.py

```diff
@@ -0,0 +1,5 @@
+import logging
+
+logging.getLogger('boto3').setLevel(logging.WARNING)
+logging.getLogger('botocore').setLevel(logging.WARNING)
+logging.getLogger('nose').setLevel(logging.WARNING)
```

Comment:
I like this, let's leave it for now, but we might want to make this more configurable at some point. It might be nice to see logs at info level when debugging, or even set it to error only in the CI pipeline.

Reply:
Yeah totally. This was mostly because `moto` would absolutely spam the console during local pytest runs.

Reply:
I figured something like that. No need to over abstract it right now, but some interesting possibilities for the future.